ambari/user-guide/manage-hosts.rst

```diff
@@ -0,0 +1,230 @@
+
+Alternatively, choose action options from the drop-down menu next to an individual component on a host. The drop-down menu shows current operation status for each component.
+
+Decommissioning Masters and Slaves
+__________________________________
+
+Decommissioning is a process that supports removing a component from the cluster. You must decommission a master or slave running on a host before removing the component or host from service. Decommissioning helps prevent potential loss of data or service disruption. Decommissioning is available for the following component types:
+
+* DataNodes
+* NodeManagers
+
+Decommissioning executes the following tasks:
+
+* For DataNodes, safely replicates the HDFS data to other DataNodes in the cluster.
+* For NodeManagers, stops accepting new job requests from the masters and stops the component.
+
+**How to Decommission a Component**
+
+To decommission a component using Ambari Web, browse Hosts to find the host FQDN on which the component resides.
+
+Using Actions, select HostsComponent Type, then choose Decommission.
+
+The UI shows "Decommissioning" status while steps process, then "Decommissioned" when complete.
+
+How to Delete a Component
+_________________________
+
+To delete a component using Ambari Web, on ``Hosts`` choose the host FQDN on which the component resides.
+
+1. In ``Components``, find a decommissioned component.
+
+2. Stop the component, if necessary.
+
+  .. Note::
+    A decommissioned slave component may restart in the decommissioned state.
+
+3. For a decommissioned component, choose Delete from the component drop-down menu.
+
+  .. Note::
+    Restarting services enables Ambari to recognize and monitor the correct number of components.
+
+  Deleting a slave component, such as a DataNode does not automatically inform a master component, such as a NameNode to remove the slave component from its exclusion list. Adding a deleted slave component back into the cluster presents the following issue; the added slave remains decommissioned from the master's perspective. Restart the master component, as a work-around.
+
+Deleting a Host from a Cluster
+______________________________
+
+Deleting a host removes the host from the cluster. Before deleting a host, you must complete the following prerequisites:
+
+* Stop all components running on the host.
+* Decommission any DataNodes running on the host.
+* Move from the host any master components, such as NameNode or ResourceManager, running on the host.
+* Turn Off Maintenance Mode, if necessary, for the host.
+
+To delete a host from a cluster:
+
+#. In Hosts, click on a host name.
+#. On the Host-Details page, select Host Actions drop-down menu.
+#. Choose Delete.
+
+If you have not completed prerequisite steps, a warning message similar to the following one appears:
+
+.. image:: /img/ambari/Unable_to_delete_host_msg.png
```

Comment:
Maybe we want to hide Ganglia Monitor from image to prevent questions on how to install it.